keylime/da/examples/redis.py

```diff
@@ -62,15 +63,20 @@ def agent_list_retrieval(self, record_prefix="auto", service="auto"):
                 agent_list.append(agent_uuid)
         return agent_list
 
-    def bulk_record_retrieval(self, record_identifier):
+    def bulk_record_retrieval(self, record_identifier, start_date=0, end_date=-1):
```

Comment:
should end_date be signalized as `-1` like here or `99999999999` like in the other module. IMHO none but there is an advantage in the second one (start <= end would be an invariant)

To represent a max value I suggest `sys.maxsize`, but there can be better candidates

Reply:
On this case, `-1` has a very specific meaning for redis sorted sets. The decision made here was to allow "implementation-specific" defaults, but override these as needed from `keylime_attest`

Reply:
I see! This `bulk_record_retrieval` method is never called from outside, it is not part of the public API, so it is natural that the defaults are different.

In that case I suggest to prefix the private methods with `_`, as they are not part of the public contract

Reply:
agreed

Reply:
fixed